KerbalStuff/common.py

```diff
@@ -198,3 +199,13 @@ def check_mod_editable(mod, abort_response=401):
     if abort_response is not None:
         abort(abort_response)
     return False
+
+def get_version_size(f):
+    if not os.path.isfile(f): return None
+
+    size = os.path.getsize(f)
+    if size < 1023: return "%d %s" % (size, ( "byte" if size == 1 else "bytes" ))
+    elif size >= 1024 and size < 1024**2: return "%3.2f KiB" % (size/1024)
+    elif size >= 1024**2 and size < 1024**3: return "%3.2f MiB" % (size/1024**2)
```

Comment:
No reason not to precalculate all the `1024**N`, so as not to spend cycles on this each time a request to the `/mod/` endpoint is made, which should be quite often.